src/components/Content/Animals/Animals.module.css

```diff
@@ -1,5 +1,81 @@
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+p{
+    font-size: 16px;
+}
+
 .animals{
-    background-color: grey ;
+    background-color: white ;
     height: 500px;
+    padding-bottom: 100px;
+
+}
+
+.animalelem{
```

Comment:
common naming convention for css is BEM - block element modifier  (http://getbem.com/)
often the names are written in kebab-case